cluster-up/version.txt

```diff
@@ -1 +1 @@
-2104301741-c950058
```

Comment:
Leftovers from a rebase that went not correct, please check it

Reply:
Removed.